MekHQ/src/mekhq/campaign/stratcon/StratconRulesManager.java

```diff
@@ -658,10 +658,10 @@ private static void setupFacilityScenario(StratconScenario scenario, StratconFac
      * Applies time-sensitive facility effects.
      */
     private static void processFacilityEffects(StratconTrackState track,
-            StratconCampaignState campaignState, boolean isMonday) {
+            StratconCampaignState campaignState, boolean isStartOfMonth) {
```

Comment:
I would prefer if this function received a date instead of a boolean, and the date itself had a function call that answers if it is the start of month.
This could enable some neat stuff, like scheduled events on facilities, etc.
Also, it would reduce the number of late changes, if we want to move from monthly back to weekly, or biweekly, it has to be changed in this funciton, and in the function that call it.

Reply:
That would be beyond the scope of this PR